unable to pass -X main.buildDate as ldflags if it contains space

likely go link command lacks proper argument passing that it loses spaces

so patch the source directly

--- caddy-0.8.3/main.go~	2016-04-26 18:28:05.000000000 +0300
+++ caddy-0.8.3/main.go	2016-05-22 14:13:18.927407255 +0300
@@ -233,7 +233,7 @@
 	appVersion = "(untracked dev build)" // inferred at startup
 	devBuild   = true                    // inferred at startup
 
-	buildDate        string // date -u
+	buildDate = "(unknown)" // date -u
 	gitTag           string // git describe --exact-match HEAD 2> /dev/null
 	gitNearestTag    string // git describe --abbrev=0 --tags HEAD
 	gitCommit        string // git rev-parse HEAD
